### ISSUE TITLE
Dropdown not work when install fresh laravel jetstream with turbo-laravel

### ISSUE THREAD

**Issue by vanthao03596** [tonysm/turbo-laravel#6]
- I followed install guide.
- Create new account in register page
- After login success, dropdown menu not work
- If i refresh page, everything work again

If i put  @livewireScripts and livewire-turbolinks.js to guest layout everthing work as expected.

**Comment by tonysm** [comment]
uhm, I think you need the latest version of Livewire (dev-master). Support for Turbo wasn't tagged yet.

**Comment by tonysm** [comment]
Updated the [documentation](https://github.com/tonysm/turbo-laravel/commit/e3df1ea0089ce57e2292b43db0fb69be0e64da8b). Change Livewire version to `dev-master` and run `composer update livewire/livewire`. If that doesn't work, feel free to re-open the issue.

Example from the demo app [here](https://github.com/tonysm/turbo-demo-app/blob/main/composer.json#L20).

**Comment by vanthao03596** [comment]
@tonysm livewire support turbo in [https://github.com/livewire/livewire/releases/tag/v2.3.6](url). If i remove app.js script from guest layout like your demo repo which mean we don't have turbo in authentication page, everthing work fine. I think alpine.js not work if we navigave with turbo after submit form.

**Comment by tonysm** [comment]
Uhm, indeed. I did remove the app.js from the guest page on my demo app. I'll take another look at it tonight.

**Comment by vanthao03596** [comment]
I found one more issue. 
If i navigate between "Profile" and "Team Settings" several time, we have error "Alpine Error: "TypeError: Cannot read property '$wire' of undefined"". 
I tried to fix and found library [alpine-turbo-drive-adapter](https://github.com/SimoTod/alpine-turbo-drive-adapter)
![1](https://user-images.githubusercontent.com/34786441/104925590-a2d77c00-59d1-11eb-864d-be19f8b19605.png)

**Comment by tonysm** [comment]
The Alpine Turbo Drive Adapter kinda makes it work, but I did have to either:

1) Add all Livewire scripts/styles to the `guest.blade.php` layout as well; or
2) Remove the `app.js` from the guest layout

Tbh, I'm not sure why the `app.js` is added to the guest layout, doesn't seem to be used anywhere in the scaffolding pages. But I do think it makes sense to use JS in guest pages, so I'm kinda torn.

I'll go with updating the `guest.blade.php` to also add the Livewire scripts there 😉

**Comment by tonysm** [comment]
Hey, @vanthao03596 

I've made the changes to also include the alpine-turbo-drive-adapter to the `turbo:install` call (see https://github.com/tonysm/turbo-laravel/releases/tag/0.0.5), can you take a look? I've also made Stimulus optional, so if you want that too, you need to pass the `--stimulus` flag.

I'm going to close, but feel free to re-open it if you think it needs to be changed.

**Comment by vanthao03596** [comment]
Great work! @tonysm

**Comment by vanthao03596** [comment]
@tonysm alpine-turbo-drive-adapter not work anymore, i have Alpine Error: "TypeError: Cannot read property '$wire' of undefined again.
if i remove alpine-turbo-drive-adapter from app.js, everything work as expected

**Comment by tonysm** [comment]
@vanthao03596 you're right. I just tested it and looks like Alpine just works now. Weird. Will investigate a bit.

**Comment by tonysm** [comment]
A little update: the problem here has to do with the Alpine+Livewire's entanglement feature together with Turbo. I've opened a PR with the breaking test in the Livewire repo [here](https://github.com/livewire/livewire/pull/3151), if anyone wants to give it a shot.

In the meantime, I shared a workaround [on Twitter](https://twitter.com/tonysmdev/status/1415860951497297921) which consists of simply disabling Turbo visits to Jetstreamp pages and using full page reloads instead. We can achieve that by adding a couple of meta tags to the page head only when visiting via Jetstream.

**resources/views/layouts/app.blade.php**
```blade
@if (config('jetstream-page'))
    <meta name="turbo-cache-control" content="no-cache">
    <meta name="turbo-visit-control" content="reload">
@endif
```

**app/Http/Middleware/JetstreamRoutes.php**
```php
<?php

namespace App\Http\Middleware;

use Closure;
use Illuminate\Http\Request;

class JetstreamRoutes
{
    public function handle(Request $request, Closure $next)
    {
        config(['jetstream-page' => true]);

        return $next($request);
    }
}
```

**app/Http/Kernel.php**
```php
<?php

namespace App\Http;

use Illuminate\Foundation\Http\Kernel as HttpKernel;

class Kernel extends HttpKernel
{
    // ...

    protected $routeMiddleware = [
        // ...
        'jetstream-routes' => \App\Http\Middleware\JetstreamRoutes::class,
    ];
}
```

**config/jetstream.php**
```php
<?php

return [
    // ...

    'middleware' => ['web', 'jetstream-routes'],

    // ...
];
```
